internal/httphandler/utils.go

```diff
@@ -164,13 +120,13 @@ func extractVerifiedQueryPayload(w http.ResponseWriter, r *http.Request, queryTy
 	case constants.GetHistoricalData:
 		version, err := getVersion(params)
```

Comment:
And here. btw, `getVersion` used multiple times

Reply:
None